pkg/scale/result.go

```diff
@@ -4,10 +4,13 @@
 package scale
 
 import (
+	"errors"
 	"fmt"
 	"reflect"
 )
 
+var ErrResultAlreadySetted = errors.New("result already has an assigned value")
```

Comment:
nit rename `ErrResultAlreadySetted` to `ErrResultAlreadySet` (past of set is set)